conftest.py

```diff
@@ -1,14 +1,50 @@
 import logging
+import os
+import pathlib
 import shutil
 
-from simple_logger.logger import get_logger
+from utilities.logger import separator, setup_logging
 
-from utilities.pytest_utils import (
-    separator,
-)
 
+LOGGER = logging.getLogger(__name__)
 BASIC_LOGGER = logging.getLogger("basic")
-LOGGER = get_logger(__name__)
+
+
+def pytest_addoption(parser):
+    log_group = parser.getgroup(name="PytestLog")
+    log_group.addoption(
+        "--pytest-log-file",
+        help="Path to pytest log file",
+        default="pytest-tests.log",
```

Comment:
may be we should give the default path to /tmp/odh-test.log

Reply:
i want to keep the same convention as pytest (default location in parent dir)